MonoGame.Framework/Media/SongCollection.cs

```diff
@@ -100,21 +138,54 @@ public SongCollection Clone()
             return sc;
         }
 
+        /// <summary>
+        /// Determines whether a <see cref="Song"/> is in the <see cref="SongCollection"/>
+        /// </summary>
+        /// <returns><see langword="true"/> if <paramref name="item"/> is found in the <see cref="SongCollection"/>;
+        /// otherwise, <see langword="false"/>.</returns>
         public bool Contains(Song item)
         {
             return innerlist.Contains(item);
         }
 
+        /// <summary>
+        /// Copies the elements of the collection to an <see cref="Array"/>,
+        /// starting at a particular <see cref="Array"/> index.
+        /// </summary>
+        /// <param name="array">
+        /// The one-dimensional <see cref="Array"/> that is the destination of the elements copied
+        /// from collection. The <see cref="Array"/> must have zero-based indexing.
+        /// </param>
+        /// <param name="arrayIndex">The zero-based index in array at which copying begins.</param>
         public void CopyTo(Song[] array, int arrayIndex)
         {
             innerlist.CopyTo(array, arrayIndex);
         }
 
-		public int IndexOf(Song item)
+        /// <summary>
+        /// Searches for the specified <see cref="Song"/> and returns the zero-based index of the
+        /// first occurence within the entire <see cref="SongCollection"/>.
+        /// </summary>
+        /// <param name="item">The <see cref="Song"/> to locate</param>
+        /// <returns>
+        /// The zero-based index of the first occurence of <paramref name="item"/> within
+        /// the entire <see cref="SongCollection"/>, if found. otherwise, -1.
+        /// </returns>
+        public int IndexOf(Song item)
         {
             return innerlist.IndexOf(item);
         }
 
+        /// <summary>
+        /// Removes the first occurrence of a <see cref="Song"/> from the <see cref="SongCollection"/>.
+        /// </summary>
+        /// <param name="item">The object to remove from the <see cref="SongCollection"/>.</param>
+        /// <returns>
+        /// <see langword="true"/> if item was successfully removed from the <see cref="SongCollection"/>;

```

Comment:
I don't think we need to include SEE references for value types like "true", simply stating "Returns True if the item was successfully removed...

Reply:
"See" references with "langword" make it look good in Intellisense, and theoretically shouldn't change DocFx docs

Reply:
Discussed this with @AristurtleDev , and while fine, it just wasn't something I've used, so I reversed my position on it :D